packages/pluggableWidgets/maps-web/src/Maps.xml

```diff
@@ -157,11 +154,11 @@
                 </property>
                 <property key="geodecodeApiKey" type="string" required="false">
                     <caption>Google maps API Key</caption>
-                    <description/>
+                    <description>If Google Maps is not used as the map provider, this property becomes optional. If you're using markers based on address, then this key is necessary for the markers to work.</description>
```

Comment:
@ConnorLand can you check whether this sentence is fine? 😄  The sentence below is an exact copy, so don't worry about it